test-cases/manager/manager-backup-and-restore-5TB-single-node.yaml

```diff
@@ -0,0 +1,28 @@
+test_duration: 4320
+
```

Comment:
General question, do we delete these backups to S3 at some point?
If so, what take care of it? (teardown? , do we have a scheduled job for it?).

Reply:
The life cycle rule defined in the bucket: https://s3.console.aws.amazon.com/s3/buckets/manager-backup-tests-us-east-1?region=us-east-1&tab=management